client/app/components/template-explorer/template-explorer.e2e.js

```diff
@@ -0,0 +1,10 @@
+describe('Component: templateExplorer', function() {
```

Comment:
if its a list, and there should be x number of items in the list, bar minimum probably also want to confirm the list is populated.